### PR TITLE
First pass at updating the daily builds table to be more informative

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

**Comment by mmitche** [comment]
@DamianEdwards @leecow @richlander This is a first stab at an updated table.  The individual repos already have large tables, so I don't really want to duplicate that much info here.

I envision that the linked core-sdk SDK pages will contain the links to coherent components

**Comment by Petermarcu** [comment]
I think this is a good start to iterate off of.

File: daily-builds.md [hunk 1]
```diff
@@ -1,11 +1,21 @@
-# .NET Core Daily Builds -- Preview Releases
+# .NET Core Daily Builds -- Servicing and future releases
 
-The .NET Core team builds several branches of the product daily. You can install these builds to try out pre-released functionality. The branches are typically either for preview or [servicing versions](daily-builds-servicing.md) of the product.
+The .NET Core team builds many branches of the product daily. You can install these builds to try out pre-released functionality.  These brnches
```
Comment (IDisposable) [inline daily-builds.md]: s/brnches/branches

File: daily-builds.md [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
-* [.NET Core Runtime 2.2 preview](https://github.com/dotnet/core-setup/blob/master/README.md#daily-builds)
\ No newline at end of file
+|Component|*.NET Core 3.0*|*.NET Core 2.2*|
+|:------:|:------:|:------:|
+|SDK|[.NET Core SDK 3.0.1xx  (Master)](https://github.com/dotnet/core-sdk/blob/master/README.md#installers-and-binaries)|[.NET Core SDK 2.2.1xx](https://github.com/dotnet/core-sdk/blob/master/README.md#installers-and-binaries)|
+|ASP.NET Core|[ASP.NET Core Shared Framework 3.0 (Master)](https://github.com/aspnet/universe#daily-builds)|[ASP.NET Core Shared Framework 2.2](https://github.com/aspnet/universe#daily-builds)|
+|Runtime|[.NET Core Runtime 3.0 (Master)](https://github.com/dotnet/core-setup/blob/master/README.md#daily-builds)|[.NET Core Runtime 2.2](https://github.com/dotnet/core-setup/blob/master/README.md#daily-builds)|
+
+## Servicing Releases
+
```
Comment (IDisposable) [inline daily-builds.md]: Insert old verbiage about servicing releases

>Servicing branches are for new patch versions of .NET Core that have not yet been released and are not yet supported. These patch versions are for previously released major/minor versions.

File: daily-builds.md [hunk 3]
```diff
@@ -1,11 +1,21 @@
-# .NET Core Daily Builds -- Preview Releases
+# .NET Core Daily Builds -- Servicing and future releases
 
-The .NET Core team builds several branches of the product daily. You can install these builds to try out pre-released functionality. The branches are typically either for preview or [servicing versions](daily-builds-servicing.md) of the product.
+The .NET Core team builds many branches of the product daily. You can install these builds to try out pre-released functionality.  These brnches
 
 Preview branches are for new major/minor versions of .NET Core that have not yet been released and are not yet supported.
```
Comment (IDisposable) [inline daily-builds.md]: Move this block **under** the `## Future Releases` section and **above** the version table... i.e. move line 5 down to line 8 (and add a blank line)

**Pull request merged**